api/src/user/user.handlers.ts

```diff
@@ -247,3 +250,167 @@ export const getUserWorksheet = async (
     data: worksheetCoursesToWorksheets(worksheets)[netId] ?? {},
   });
 };
+
+const AddWorksheetSchema = z.object({
+  action: z.literal('add'),
+  season: z.string().transform((val) => parseInt(val, 10)),
```

Comment:
Can we take a "name" parameter? I'm thinking the following UI: the bottom of the worksheet names list is a "+", and when you click it, it turns into a text box, which by default is filled with "New worksheet", and you can edit this text before you even request adding it. This means the API doesn't have to hardcode this default name.